### ISSUE TITLE
CefSharp.WinForms 81.3.100 sample application crashes

### ISSUE THREAD

**Issue by sanjaykulkarni04** [cefsharp/CefSharp#3150]
What version of the product are you using?
Nuget CefSharp.WinForms 81.3.100
What architecture x86 or x64?
 x64
On what operating system?
Win10
What is the .net framework?
4.5.2
Are you using WinForms, WPF or OffScreen?
WinForms
What steps will reproduce the problem?
Below is the sample code to reproduce issue
public partial class Form1 : Form
    {
        public ChromiumWebBrowser chromeBrowser;
        public Form1()
        {
            InitializeComponent();
            InitializeChromium();
        }
        public void InitializeChromium()
        {
            CefSettings settings = new CefSettings();
            // Initialize cef with the provided settings
            Cef.Initialize(settings);
            // Create a browser component
            chromeBrowser = new ChromiumWebBrowser("http://google.com");
            // Add it to the form and fill it to the form window.
            this.Controls.Add(chromeBrowser);
            chromeBrowser.Dock = DockStyle.Fill;   
        }        
        private void Form1_FormClosing(object sender, FormClosingEventArgs e)
        {
            Cef.Shutdown();
        }      
    }
What is the expected output? What do you see instead?
Winforms not responding/crash and no errors are seen in the debug.log file. Even sometimes machine not responding and need to do a hard reboot. Sometimes CPU/Memory utilization shoots up to 100%.
Please provide any additional information below.
**_App.config file -_**
<?xml version="1.0" encoding="utf-8" ?>
<configuration>
    <startup> 
        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.2" />
    </startup>
	<runtime>
		<assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
			<probing privatePath="x86"/>
		</assemblyBinding>
	</runtime>
</configuration>

I followed below steps for sample application test -

https://ourcodeworld.com/articles/read/173/how-to-use-cefsharp-chromium-embedded-framework-csharp-in-a-winforms-application

**Comment by amaitland** [comment]
Does the problem occur with the minimum example? 

https://github.com/cefsharp/CefSharp.MinimalExample

Please test with the CEF Sample application as detailed in the bug report. 

https://github.com/cefsharp/CefSharp/blob/master/.github/ISSUE_TEMPLATE/bug_report.md#bug-report

You should also check the log file https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file

Based on the sample it's unlikely a bug in CefSharp.

**Comment by sanjaykulkarni04** [comment]
Yes, the same problem with the minimum example as well. https://github.com/cefsharp/CefSharp.MinimalExample

Do you need dump files?

Below details from **debug.log** file :
[0610/161157.003:INFO:crash_reporting.cc(219)] Crash reporting enabled for process: browser
[0610/161157.338:INFO:crash_reporting.cc(219)] Crash reporting enabled for process: gpu-process
[0610/161157.645:INFO:crash_reporting.cc(219)] Crash reporting enabled for process: utility
[0610/161158.618:INFO:crash_reporting.cc(219)] Crash reporting enabled for process: utility
[0610/161358.332:INFO:crash_reporting.cc(219)] Crash reporting enabled for process: gpu-process
[0610/162126.427:INFO:crash_reporting.cc(219)] Crash reporting enabled for process: browser
[0610/162126.814:INFO:crash_reporting.cc(219)] Crash reporting enabled for process: gpu-process
[0610/162127.395:INFO:crash_reporting.cc(219)] Crash reporting enabled for process: utility
[0610/162128.528:INFO:crash_reporting.cc(219)] Crash reporting enabled for process: utility

**Comment by amaitland** [comment]
Please test with the CEF Sample application.

**Comment by amaitland** [comment]
See also https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#loading-native-symbols-for-easier-diagnosis

**Comment by sanjaykulkarni04** [comment]
Thanks, Alex. I will go through these steps and let you know.
Meanwhile, I am attaching a chrome GPU settings report from my machine.

[Chrome GPU settings.docx](https://github.com/cefsharp/CefSharp/files/4763479/Chrome.GPU.settings.docx)

**Comment by sanjaykulkarni04** [comment]
Hi Alex,

In **debug.log** I can see error messages - 

[0610/220839.118:ERROR:cache_util_win.cc(21)] Unable to move the cache: Access is denied. (0x5)
[0610/220839.127:ERROR:cache_util.cc(138)] Unable to move cache folder C:\Users\skulkarni\AppData\Local\CefSharp\Cache\GPUCache to C:\Users\skulkarni\AppData\Local\CefSharp\Cache\old_GPUCache_000
[0610/220839.145:ERROR:disk_cache.cc(184)] Unable to create cache
[0610/220839.145:ERROR:shader_disk_cache.cc(606)] Shader Cache Creation failed: -2
